src/WebJobs.Script/Scale/FunctionsScaleManager.cs

```diff
@@ -37,11 +37,11 @@ public FunctionsScaleManager(IScaleMonitorManager monitorManager, IScaleMetricsR
         /// </summary>
         /// <param name="context">The context to use for the scale decision.</param>
         /// <returns>The scale vote.</returns>
-        public virtual async Task<ScaleStatusResult> GetScaleStatusAsync(ScaleStatusContext context)
+        public virtual async Task<int> GetScaleStatusAsync(ScaleStatusContext context)
```

Comment:
I am more of a fan of the previous explicit result type. The `int` is a bit vague. What does it mean when I return something outside of -1, 0, 1? What unit does this int represent? Maybe just get rid of the wrapper object and use `ScaleVote` enum directly? Unless you do want to support values outside of -1, 0, 1. If that is true, can we think about what the unit represents? Maybe documentation is all that is needed.

Reply:
I was also curious why we're getting rid of `ScaleStatusResult` for `int`